apps/coscrad-frontend/src/store/slices/resources/transcribed-audio/types/TranscribedAudioSliceState.ts

```diff
@@ -0,0 +1,4 @@
+import { IIndexQueryResult, ITranscribedAudioViewModel } from '@coscrad/api-interfaces';
+import { ILoadable } from '../../../interfaces/loadable.interface';
+
+export type TranscribedAudioSliceState = ILoadable<IIndexQueryResult<ITranscribedAudioViewModel>>;
```

Comment:
We may want to alias `IIndexQueryResult<ITransribedAudioViewModel>`; it shows up everywhere.

Reply:
I've done this in more recent slices, but will not fix the vestiges until we have occasion to revisit the code for something productive.